errors/index.js

```diff
@@ -1,3 +1,15 @@
-module.exports.serverErrors = (error, request, respond, next) => {
-    respond.status(500).send({message: 'Internal Server Error'});
+module.exports.serverErrors = (error, request, response, next) => {
+    response.status(500).send({message: 'Internal Server Error'});
```

Comment:
It's usually best practice to have this error handler at the bottom of the file. This is because it's the least likely to be picked up, so we'd check the other first 👍

Reply:
It happened to me where I invoke the middleware functions, and I realised it, I totally forgot that it goes from top to bottom and check conditionally, however I think app.js file handles it now, the error/index.js file just declares the functions.